web/modules/custom/portland/modules/portland_address_verifier/src/Plugin/WebformElement/PortlandAddressVerifier.php

```diff
@@ -124,6 +87,8 @@ public function prepare(array &$element, WebformSubmissionInterface $webform_sub
     $machine_name = "edit-" . $key . "--wrapper";
     $machine_name = str_replace("_", "-", $machine_name);
 
+    $element['#attached']['drupalSettings']['webform']['portland_address_verifier'][$machine_name]['verification_required'] = !empty($element['#location_verification_status__required']);
```

Comment:
This determines whether address verification is required and passes it to the client.